app/helpers/filter_navigation/people.rb

```diff
@@ -35,13 +35,20 @@ def match
     def init_kind_filter_names
       @kind_filter_names = {}
       Role::Kinds.each do |kind|
-        @kind_filter_names[kind] = I18n.t("activerecord.attributes.role.class.kind.#{kind}.other")
+        archived = 'archived' if group.archived?
+        translation_path = ['activerecord.attributes.role.class.kind',
+                            archived,
+                            kind,
+                            'other'].join('.')
+        @kind_filter_names[kind] = I18n.t(translation_path)
```

Comment:
Macht es das nicht nötig, in allen Wagons die Übersetzungen anzupassen? Würde es reichen wenn du eine einzige Übersetzung `%{role_kind} (archiviert)` mit Platzhalter definierst, und dann folgendes verwendest:
```ruby
I18n.t('activerecord.attributes.role.archived',
       role_kind: I18n.t('activerecord.attributes.role.class.kind.#{kind}.other'))
```
Damit könnte man sogar dem Gender-Problem ausweichen.